Starting job at Mon Feb 24 01:52:39 PM CET 2025
Running Filter_by_part.py for subdirectory 22012, part 27
2025-02-24 13:52:39 - INFO - Event filtering process starts...
2025-02-24 13:52:39 - INFO - Initialized PureNeutrinoEventFilter filter for 22012/27
2025-02-24 13:52:39 - INFO - Loaded 14 event specifier files from /lustre/hpc/project/icecube/HE_Nu_Aske_Oct2024/clean_events_dict/22012/22012/reduced
2025-02-24 13:52:39 - INFO - Selected 0 CSV files covering event range [14444, 14999]
2025-02-24 13:52:39 - WARNING - No matching CSV files found for 22012/27
2025-02-24 13:52:39 - INFO - Loaded 0 valid (RunID, EventID) pairs from CSV files
2025-02-24 13:52:39 - WARNING - No valid events in 22012/27. Skipping filtering.
2025-02-24 13:52:39 - INFO - Initialized CCFilter filter for 22012/27
2025-02-24 13:52:39 - INFO - Extracted 25657 valid events which are CC interactions.
2025-02-24 13:52:39 - INFO - Initialized ContainmentFilter filter for 22012/27
2025-02-24 13:52:40 - INFO - Extracted 2245 valid events within the specified containment region.
2025-02-24 13:52:40 - ERROR - An error occurred: 'PureNeutrinoEventFilter' object has no attribute 'valid_event_nos'
Traceback (most recent call last):
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 69, in <module>
    main()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 65, in main
    run()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 52, in run
    filter_manager = EventFilterManager(
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilterManager.py", line 19, in __init__
    self.filter_obj = self._build_Filter_object()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilterManager.py", line 54, in _build_Filter_object
    valid_event_nos = self._synthesize_valid_event_nos()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilterManager.py", line 78, in _synthesize_valid_event_nos
    event_sets = {alias: f.get_valid_event_nos() for alias, f in self.filter_instances.items()}
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilterManager.py", line 78, in <dictcomp>
    event_sets = {alias: f.get_valid_event_nos() for alias, f in self.filter_instances.items()}
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilter.py", line 36, in get_valid_event_nos
    return self.valid_event_nos
AttributeError: 'PureNeutrinoEventFilter' object has no attribute 'valid_event_nos'
Job completed at Mon Feb 24 01:52:40 PM CET 2025
